public/app/features/trails/DataTrailsHistory.test.tsx

```diff
@@ -22,7 +22,7 @@ describe('DataTrailsHistory', () => {
       {
         name: 'from history',
         input: { from: '2024-07-22T18:30:00.000Z', to: '2024-07-22T19:30:00.000Z' },
-        expected: '2024-07-22 12:30:00 - 2024-07-22 13:30:00',
+        expected: '2024-07-22 13:30:00 - 2024-07-22 14:30:00',
```

Comment:
I've fixed this issue in a PR https://github.com/grafana/grafana/pull/93096 
It has a bug because of the timezones and day light saving. 
I'd recommend merging `main` into your branch

Reply:
Can you merge `main` and have the changes in PR #93096 ?

Reply:
Fixed!